src/style.css

```diff
@@ -0,0 +1,118 @@
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  font-family: roboto, sans-serif;
+  width: 200px;
+  height: 410px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__image {
+  padding: 0 3px;
+  display: flex;
+  justify-content: center;
+}
+
+.card__image_picture {
+  width: 160px;
+  height: 134px;
+}
+
+.card__specifications {
+  margin-top: 40px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.card__code {
+  margin-top: 4px;
+  font-size: 10px;
+  font-weight: 400px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.card__review_stars {
+  display: flex;
+}
+
+.card__review_stars_star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.card__review_stars :nth-child(-n+4) {
+  background: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
reuse it from previous task
check how the import works
https://developer.mozilla.org/en-US/docs/Web/CSS/@import